enterprise/models.py

```diff
@@ -1456,6 +1456,10 @@ class EnterpriseCustomerIdentityProvider(TimeStampedModel):
         unique=True,
         help_text="Slug field containing a unique identifier for the identity provider.",
     )
+    default = models.BooleanField(
```

Comment:
Wouldn't we need to make any code changes in the enterprise SSO login flow to make decisions based on this flag?

Reply:
@sameenfatima78 That is part of another ticket.